base/linalg/arnoldi.jl

```diff
@@ -372,8 +372,10 @@ iterations derived from [`eigs`](@ref).
 **Outputs**
 
 * `svd`: An `SVD` object containing the left singular vectors, the requested values, and the
-  right singular vectors. If `ritzvec = false`, the left and right singular vectors will be
-  empty.
+right singular vectors. If `ritzvec = false`, the left and right singular vectors will be empty.
```

Comment:
does this render okay or does it need these lines to be indented with the rest of the bullet?